commands/info.js

```diff
@@ -0,0 +1,172 @@
+/* eslint-env node, es6 */
+const path = require('path');
+const fs = require('fs');
+const chalk = require('chalk');
+const spawn = require('cross-spawn');
+const minimist = require('minimist');
+const resolveSync = require('resolve').sync;
+
+function displayHelp() {
+	let e = 'node ' + path.relative(process.cwd(), __filename);
+	if (require.main !== module) e = 'enact info';
```

Comment:
I noticed a few of our commands don't use the format.  `template` and `eject`, for example.

Reply:
Forgivable, but will correct for consistency. It's mainly to cover post-ejected state and a few other edge/unusual cases where commands are executed directly, to have help info match.

Reply:
We don't _have_ to fix this, was just pointing it out in case it was actually important.